patches/server/0224-Use-AsyncAppender-to-keep-logging-IO-off-main-thread.patch

```diff
@@ -16,6 +16,46 @@ index 34a0d6b54a15c8aa0c706541316c5d448e3d94b9..76a8db5ff623b0a3b83a0861f642da67
  
      runtimeOnly("org.apache.maven:maven-resolver-provider:3.8.1")
      runtimeOnly("org.apache.maven.resolver:maven-resolver-connector-basic:1.7.0")
+diff --git a/src/main/java/net/minecraft/server/Main.java b/src/main/java/net/minecraft/server/Main.java
+index fb0b3c5770f66cc3590f5ac4e690a33cb6179be3..1537e1b54564295acf0c0466156b4a1f63919922 100644
+--- a/src/main/java/net/minecraft/server/Main.java
++++ b/src/main/java/net/minecraft/server/Main.java
+@@ -112,6 +112,8 @@ public class Main {
+ 
+             if (optionset.has("initSettings")) { // CraftBukkit
+                 Main.LOGGER.info("Initialized '{}' and '{}'", path.toAbsolutePath(), path1.toAbsolutePath());
++                LogManager.shutdown(); // Paper - workaround until Log4J is updated.
++                Process.exit(0); // Paper - workaround until Log4J is updated.
```

Comment:
Maybe my brain is failing me, but what method is `Process#exit`? This does not compile for me locally and I haven't heard of that method either 😅 

Was this supposed to be `System#exit` ?

Reply:
Whoops, that's very wrong 😂 . I have no idea how I managed to miss that.

Reply:
Fixed now!